src/main/java/org/broadinstitute/consent/http/models/Institution.java

```diff
@@ -85,4 +102,11 @@ public Date getUpdateDate() {
   public Integer getUpdateUser() {
     return updateUser;
   }
+
+  @Override
+  public boolean equals(Object institution) {
+    if (institution == this) return true;
+    if (institution == null || institution.getClass() != getClass()) return false;
+    return (((Institution)institution).getId() == getId());
```

Comment:
For consistency, we should use `EqualsBuilder`. If we need to add something else to determine equality, it will be trivial to add an append line.
```suggestion
    Institution other = (Institution) obj;
    return new EqualsBuilder()
          .append(id, other.getId())
          .isEquals();
```